IaC/PnP/monitoring/aks/online/aks_secure_baseline/ngsa-memory/ngsa-memory.yaml

```diff
@@ -0,0 +1,107 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    name: ngsa
+  name: ngsa
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ngsa-memory
+  namespace: ngsa
+# We should label our app with proper k8s label
+  labels:
+    app.kubernetes.io/name: ngsa-memory
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: ngsa-memory
+  template:
+    metadata:
+      labels:
+        app: ngsa-memory
```

Comment:
version and deploy are inserted by AutoGitOps and aren't necessary. Probably less confusing to remove.

Reply:
I'll remove them when I integrate this into the gitops architecture. For now, this yaml file can be used to deploy ngsa manually or using a flux CD instance.